source/versioned-api/versioned-api.rst

```diff
@@ -90,7 +90,7 @@ declare an API version:
 Drivers SHOULD group the ``serverApi`` option with other similar client options
 like ``autoEncryptionOpts``. Drivers MUST NOT allow specification of any
 versioned API options via the connection string. See the
-`design rationale <_rationale_no_uri_options>`_ for more details.
+`Design Rationale`_ for more details.
```

Comment:
Looks like this dates back to the original spec commit: https://github.com/mongodb/specifications/commit/784857f548747ee6f894a5b6f7b6e5bb89103709#diff-d81feb8a94e1a811734074bde8dff007c7872dde8117e6ccdc06c6eedcce268dR93